README.md

```diff
@@ -82,6 +84,52 @@ When this element is rendered, the DOM will look something like the following:
 
 This is the utility that web components provide when using Custom Elements and the Shadow DOM.
 
+Skate also allows **turning off Shadow DOM** if you don't wanna use it for various particular reasons. You can turn it off via `get renderRoot()` override:
+
+> NOTE: by turning off Shadow DOM you cannot use <slot/> content projection anymore by default, further tweaks needs to be applied
+
+```js
+import { withComponent, props } from 'skatejs';
+
+// define base class withouth Shadow DOM
+class NoShadowComponent = class extends withComponent() {
```

Comment:
just thinking out loud here, maybe we can add `withoutShadomDOM()` mixin to skate API, so then user can do just
`const Component = withComponent(withoutShadomDOM(withCustomRenderer()))` 
WDYT ?

Reply:
I don't think we need an extra proto chain just for that and shadow dom is only specific to the `withRenderer` mixin. I say just leave it, but happy for others to weight in. @chrisdarroch may have an opinion here.

Reply:
I think it's reasonable for people who don't want the shadow DOM to opt out themselves. Having the override point should be sufficient. My general thoughts are: make the most common thing easy, make the less common thing possible; the people who know what they're doing will manage the additional complexity.